spec/actions/lookupObject.spec.js

```diff
@@ -59,6 +59,44 @@ describe('Lookup Object (at most 1) module: getLookupFieldsModel', () => {
   it('Retrieves the list of unique fields of specified sobject', testUniqueFields.bind(null, 'Document', metaModelDocumentReply));
 });
 
+describe('Lookup Object (at most 1) module: getLinkedObjectsModel', () => {
+  async function testLinkedObjects(object, getMetaModelReply) {
+    const sfScope = nock(testCommon.configuration.oauth.instance_url)
+      .get(`/services/data/v${common.globalConsts.SALESFORCE_API_VERSION}/sobjects/${object}/describe`)
+      .reply(200, getMetaModelReply);
+
+    nock(testCommon.refresh_token.url)
+      .post('')
+      .reply(200, testCommon.refresh_token.response);
+
+    const expectedResult = {};
+    const parents = getMetaModelReply.fields.filter(field => field.type === 'reference')
```

Comment:
Correct  me if I am wrong, but it seems like you are using in tests same logic to code. It shouldn't be like that - you need to simply it